### PR TITLE
Add Travis CI instructions

### PR DESCRIPTION
**Pull request by jfbastien**
I just set up Travis CI for my repo: https://github.com/jfbastien/papers/pull/32
Adding instructions here so others don't have to figure it out.

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -35,8 +35,35 @@ $ sudo pip install lxml --upgrade
...
+python:
+  - "2.7"
+install:
+  - pip install pygments lxml setuptools --upgrade
+  - git clone https://github.com/tabatkins/bikeshed.git
+  - pip install --editable $PWD/bikeshed
+  - bikeshed update
+script:
+  # Invoke bikeshed here, at your own leisure. E.g.:
+  - find ./ -name "*.bs" -type f | xargs -I{} -t -n1 bikeshed spec {}
```
Comment (tabatkins) [inline docs/install.md]: This seems like it's just finding the first .bs file in the folder and then passing it to bikeshed, yes? No reason to be overcomplicated, then - just calling `bikeshed` with no additional arguments will do the exact same thing.

File: docs/install.md [hunk 2]
```diff
@@ -35,8 +35,35 @@ $ sudo pip install lxml --upgrade
 
 That'll spew a lot of console trash, but don't worry about it.
 
+In some installations you'll need to also upgrade `setuptools` for `html5lib`'s entertainement:
```
Comment (tabatkins) [inline docs/install.md]: "entertainment"

File: docs/install.md (hunk 1 above)
Comment (jfbastien) [inline docs/install.md]: Oh, I didn't know that.

File: docs/install.md (hunk 2 above)
Comment (jfbastien) [inline docs/install.md]: Done.

**Comment by jfbastien** [comment]
Comments addressed.

**Pull request merged**